blocked-edges/4.12.0-rc.0-AWSOldBootImage.yaml

```diff
@@ -0,0 +1,21 @@
+to: 4.12.0-rc.0
+from: 4[.]11[.].*
+url: https://issues.redhat.com/browse/COS-1942
+name: AWSOldBootImages
+message: |-
+  4.2 AWS boot images are not compatible with 4.12.0-rc.0, and machines created with them will fail to become nodes.
```

Comment:
Can we hint about that it's safe to upgrade if the admin is sure that the cluster is younger than 4.2?

Reply:
we link to COS-1942, which covers that assessment in the impact statement, but :shrug: I'm fine inlining more text here if you like.  What extra words were you thinking of?

Reply:
COS-1942 is a wall of text, I'm looking for a fast and easy reassurance for the majority (I'd expect?) case. Let me try to whip up a suggestion.

Reply:
```suggestion
  4.2 AWS boot images are not compatible with 4.12.0-rc.0, and machines created with them will fail to become nodes. It is safe to upgrade if a cluster was installed as version 4.3 or later.
```

Reply:
Not a blocker though, feel free to say that it's not necessary ;)

Reply:
I've added:

> This risk does not apply if a cluster is not on AWS, was installed as version 4.3 or later, or otherwise uses 4.3 or later boot images.

which diverges from your suggestion because:

* "It is safe to upgrade" -> "This risk does not apply", because there may be other risks that make updates unsafe.
* Adding a platform qualifier, in case the "AWS boot images" in the first sentence is too subtle for anyone on GCP, vSphere, etc.
* Adding a boot-image mention, because born-on-4.2 isn't a problem if you've subsequently bumped your boot images.

How does that look?

Reply:
![image](https://user-images.githubusercontent.com/712614/208081788-34c37907-5b68-4ba6-a6b3-a34a8846bfe3.png)